tests/test_field_data.py

```diff
@@ -375,6 +529,120 @@ def test_field_data_objects_3d(new_solver_session) -> None:
     assert all(path_lines_data["cold-inlet"]["lines"][100] == [100, 101])
 
 
+@pytest.mark.fluent_version(">=23.2")
+def test_field_data_objects_3d(new_solver_session) -> None:
+    solver = new_solver_session
+    import_file_name = examples.download_file(
+        "mixing_elbow.msh.h5", "pyfluent/mixing_elbow"
+    )
+
+    field_data = solver.fields.field_data
+
+    assert [] == field_data.get_scalar_field_data.field_name.allowed_values()
+
+    solver.file.read(file_type="case", file_name=import_file_name)
+
+    allowed_args_no_init = field_data.get_scalar_field_data.field_name.allowed_values()
+    assert len(allowed_args_no_init) != 0
+
+    assert not field_data.is_data_valid()
+
+    solver.solution.initialization.hybrid_initialize()
+
+    assert field_data.is_data_valid()
+
+    # Absolute Pressure data over the cold-inlet (surface_id=3)
+    sc1 = ScalarFieldData(field_name="absolute-pressure", surfaces=["cold-inlet"])
+    abs_press_data = field_data.get_field_data(sc1)
+
+    assert abs_press_data["cold-inlet"].shape == (241,)
+    assert abs_press_data["cold-inlet"][120] == 101325.0
+
+    su1 = SurfaceFieldData(
+        data_types=[SurfaceDataType.Vertices], surfaces=["cold-inlet"]
+    )
+    vertices_data = field_data.get_field_data(su1)
+    assert vertices_data["cold-inlet"][SurfaceDataType.Vertices].shape == (241, 3)
+    assert (
+        round(float(vertices_data["cold-inlet"][SurfaceDataType.Vertices][5][0]), 2)
+        == -0.2
+    )
+
+    su2 = SurfaceFieldData(
```

Comment:
@prmukherj If this is a request type then please append the name with `Request`.

Reply:
Actually it can again be re-used for fetching the data as well. That's why this naming.

Reply:
@prmukherj It's always a request.

Reply:
got it. Makes sense. Will update.